tests/__tests__/sitka_utils.test.ts

```diff
@@ -1,13 +1,13 @@
-import { sitka } from "../sitka-test"
+import { sitkaFactory, sitka } from "../sitka-test"
 import { createAppStore } from "../../src/sitka"
 import rewire from "rewire"
 
 const utils = rewire("../../dist/sitka.js")
 
-const { text: textModule } = sitka.getModules()
-
 describe("Sitka Util Functions", () => {
   test(`createAppStore returns Redux store`, () => {
+    // todo: using sitka factory results in referenceError: alert is not defined
```

Comment:
related to issue testing for - running test passes but results in referenceError: alert not defined?

this happens when switching this test to use sitkaFactory